autoware_launch/config/planning/preset/default_preset.yaml

```diff
@@ -120,7 +120,7 @@ launch:
 
   - arg:
       name: launch_surround_obstacle_checker
-      default: "false"
+      default: "true"
```

Comment:
Is this intentional change?

Reply:
Yes. But I rebased this branch for awf/main.